src/components/Schedule/Schedule.css

```diff
@@ -11,6 +10,9 @@
 	--disabled-text-color-btn: rgba(147, 147, 147, 0.659);
 	--background-color-hover-btn: #f094bc;
 
+	--coolvetica-font-family: Coolvetica;
+	--rubik-font-family: Rubik;
```

Comment:
These should probably be declared in https://github.com/HacktheHill/2023.hackthehill.com/blob/main/src/components/App/App.css and used everywhere, but that's not in the scope of this pull request. We can leave this file like this for now.